ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -658,6 +668,26 @@ community_lists:
     action: "< action as string >"
 ```
 
+The improved data model has a better design documented below:
+
+```yaml
+ip_community_lists:
+  - name: "<ip community list name as string>"  # mandatory
+    entries:
+      - action: "< permit | deny >"  # required
+        # community_values and regexp MUST not be configured together in the same entry
+        # possible community strings are (case insensitive):
+        # - GSHUT
+        # - internet
+        # - local-as
+        # - no-advertise
+        # - no-export
+        # - <1-4294967040>
+        # - aa:nn
+        community_values: [ "< a_community as string >", "< another_community as string >", ... ]  # optional, if defined - standard community list will be configured
```

Comment:
!! Not required but a suggestion.

Please use the yaml way of defining list:

```
community_values:
  - "< a_community as string >"
  - "< another_community as string >"
```

Reply:
@ClausHolbechArista to suggest what is more readable. Bothe are YAMLs, and I'm personally fine with both. Just prefer a more compact way in this specific case.

Reply:
As there is no other opinion, going to leave it as is. IMO, it's more readable.

Reply:
That's fine with me.